### DISCUSSION TITLE
Lets compare the UI

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#11069] [General]
While thinking about ready or not for a beta version (see #10922) I took the time to compare the current `release` with current `master`  and made some comments along. 
Maybe its a good idea to review old and new look to see if we lost something on the way or made something worse.

| Release | Master | Comment Lineflyer |
|-------|------|---|
| ![Screenshot_20210703-211635](https://user-images.githubusercontent.com/949669/124364929-14844900-dc45-11eb-9553-ae8b235f1fcb.png) | ![Screenshot_20210703-212106](https://user-images.githubusercontent.com/949669/124364934-1f3ede00-dc45-11eb-9bc7-2266fb8a0cc4.png) |  The new version looks way better. Do we want to restore the accent color line below the action bar? | 
|  ![Screenshot_20210703-211812](https://user-images.githubusercontent.com/949669/124364971-5f9e5c00-dc45-11eb-9d1d-fa200b4416e9.png) |  ![Screenshot_20210703-212114](https://user-images.githubusercontent.com/949669/124364976-675e0080-dc45-11eb-8926-a0b420f96b17.png) |  I like the increased font size. Arrows are cut off in both versions | 
|![Screenshot_20210703-211521](https://user-images.githubusercontent.com/949669/124365052-dcc9d100-dc45-11eb-96dc-f27eb762a37d.png)| ![Screenshot_20210703-212134](https://user-images.githubusercontent.com/949669/124365062-e81cfc80-dc45-11eb-97f3-22eadfe777d9.png)| Positive: The larger font size. ~Negative: The info listing looks crowded. Maybe we should use different colors for category and content as we did in the old UI?~ (Done with #11071) |
|![Screenshot_20210703-211526](https://user-images.githubusercontent.com/949669/124365104-2dd9c500-dc46-11eb-8de5-6ebca3ecc4d6.png)| ![Screenshot_20210703-212145](https://user-images.githubusercontent.com/949669/124365106-36320000-dc46-11eb-898f-ee18375fade5.png)| ~IMHO the new UI lost some structure which we had in the old UI by using different sizes and colors for the text. Side note: Why do we waste so much spaces on top before the first waypoint is shown?~ (Done with #11071) Also in the new UI we could place the two buttons more narrow to each other. |
|![Screenshot_20210703-211530](https://user-images.githubusercontent.com/949669/124365202-be180a00-dc46-11eb-85a0-9aa12814af40.png)|![Screenshot_20210703-212137](https://user-images.githubusercontent.com/949669/124365212-c5d7ae80-dc46-11eb-804b-ab18edd59a57.png)| Larger fonts, very good. |

Please feel free to comment and/or also compare some old and new screenshots, discuss and maybe extract some ToDo into issues.

**Comment by moving-bits** [comment]
Thanks @Lineflyer for making this comparison pictures. (Reminds me of a similar collection when preparing for target SDK 26 :-))
I'll start working on cache info / color differentiation and waypoints detail page.

**Reply by Lineflyer** [reply]
Feel free to transfer these items into issues to keep the overview.
I just started here as a broader discussion to break it down into issues (if others agree to my comments).

**Reply by moving-bits** [reply]
I've prepared #11071 to tackle three of the issues and updated the text above.

**Reply by fm-sys** [reply]
> Do we want to restore the accent color line below the action bar?

IIRC we had this discussion and explicitly decided to remove the line for a more consistent look. BTW, will get superfluous with the bottom navigation anyway ;-)

> Also in the new UI we could place the two buttons more narrow to each other.

I think the padding between the buttons is just fine and results in a nice and clean look. The unnecessary space between the buttons and the list view got already removed...

**Comment by Lineflyer** [comment]
Just another small comparison between 2021.06.23 and current `release` (but including my pending PRs for direction arrow and fav count size) on same device with same font settings:

![image](https://user-images.githubusercontent.com/949669/125344726-efd05580-e357-11eb-9f12-b193571887a4.png)

Looks pretty OK: A little bit larger size but nothing out of the ordinary.

**Reply by fm-sys** [reply]
On the screenshots, it seems like the GCVote is missing on our new UI? Or was there a difference in the way the screenshots where created?

**Reply by Lineflyer** [reply]
Interesting and good catch indeed. 
On both versions I did not enter my GCVote credentials, but this is no problem to request votes.
Not sure if our old nearby search requested GCVote but the new search does not? That could explain. The existing GCVote for GC7Q8TF is due to the fact, that the cache was opened already and thus the vote still existing.

@eddiemuc Did you accidentally drop GCVOte retrieval on (nearby) search?

**Reply by SammysHP** [reply]
Also the distance is known in the second screenshot.

**Reply by fm-sys** [reply]
> Also the distance is known in the second screenshot.

...which is IMHO a big step forward 👍

**Reply by Lineflyer** [reply]
Thats an advantage of the new search. It provides coords to BM.
